### PR TITLE
Update deploy actions

### PR DESCRIPTION
**Pull request by zwolf**

### CODE REVIEW

File: .github/workflows/deploy_production.yml [hunk 1]
```diff
@@ -0,0 +1,90 @@
+name: Deploy ASM to Production
+
+on:
+    # Run this workflow on push to master (incl PR merges)
+    push:
+        branches: [ master ]
```
Comment (srallen) [inline .github/workflows/deploy_production.yml:6]: For consistency, should we make this deploy on production-release tag?

**Review by srallen — COMMENTED** [review]
Just a question on the prod deploy trigger

File: .github/workflows/deploy_production.yml (hunk 1 above)
Comment (zwolf) [reply .github/workflows/deploy_production.yml:6]: oops, must have copied from the staging template

**Review by srallen — APPROVED** [review]

File: .github/workflows/deploy_production.yml [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+            az storage blob upload-batch \
+              --account-name zooniversestatic \
+              --content-cache-control 'public, immutable, max-age=604800' \
+              --destination '$web/www.antislaverymanuscripts.org' \
+              --source ./dist
+            az storage blob upload \
+              --account-name zooniversestatic \
+              --content-cache-control 'public, max-age=60' \
+              --container-name '$web' \
+              --name 'antislaverymanuscripts.org/index.html' \
```
Comment (srallen) [inline .github/workflows/deploy_production.yml:42]: Should this have `www` in the name?

Comment (zwolf) [reply .github/workflows/deploy_production.yml:42]: 💯

**Pull request merged**